public/app/features/auth-config/FieldRenderer.tsx

```diff
@@ -42,6 +42,13 @@ export const FieldRenderer = ({
     }
   }, [unregister, name, parentValue, isDependantField]);
 
+  useEffect(() => {
+    if (fieldData.defaultValue) {
+      setValue(name, fieldData.defaultValue.value);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
```

Comment:
nit: why not move this `useEffect` to the one that's right above it?

Reply:
The `useEffect` above has specific dependencies and this one should run on each render.